.actions/requirements.txt

```
@@ -1,3 +1,3 @@
-fire
+jsonargparse >= 4.15.0
```

nit: usually we don't use spaces

```suggestion
jsonargparse>=4.15.0
```